pages/1.9/release-notes/1.9.8/index.md

```diff
@@ -0,0 +1,235 @@
+
+- DC/OS CLI highlights:
+
+  - Support single sign-on authentication via OpenID Connect and SAML 2.0 against the DC/OS IAM.
+  - Support authentication with service account credentials.
+
+- Introduce various secrets improvements (for more information, see the [secrets documentation](/1.9/security/ent/secrets/)).
+- Security hardening across the platform, including Mesos, Marathon, and Admin Router.
+
+## Developer Services
+
+- Jenkins
+
+    - The Jenkins DC/OS service will now work with DC/OS clusters in strict mode. <!-- (Enterprise Only) -->
+    - Marathon plugin now supports service accounts, allowing easy automated and secure deploys to DC/OS clusters. <!-- (Enterprise Only) -->
+
+## Other Improvements
+
+### DC/OS Internals
+
+- Update DC/OS internal JDK to 8u112 for security [fixes](http://www.oracle.com/technetwork/java/javase/2col/8u112-bugfixes-3124974.html).
+- Update DC/OS internal Python from 3.4 to 3.5.
+- The `dcos_generate_config.ee.sh --aws-cloudformation` command will now determine the region of the s3 bucket automatically, preventing region mistakes.
+- Added `dcos-shell` which activates the DC/OS environment for running other DC/OS command line tools.
+- Added the `reset-superuser` script which attempts to create or restore superuser privileges for a given DC/OS user. <!-- Enterprise -->
+
+### Expanded OS Support
+
+- If you install DC/OS 1.9 using the [GUI](/1.9/installing/ent/custom/gui/) or [CLI](/1.9/installing/ent/custom/cli/) installation methods, your system will be automatically upgraded to [the latest version of CentOS](https://access.redhat.com/documentation/en/red-hat-enterprise-linux/).
+- CoreOS [1235.12.0](https://coreos.com/releases/#1235.12.0).
+
+### Expanded Docker Engine Support
+
+- Docker 1.12 and 1.13 are now [supported](/1.9/installing/ent/custom/system-requirements/). Docker 1.13 is the default version.
+
+### Upgrades
+
+Improved upgrade tooling and experience for on-premise installations. Upgrades now use internal DC/OS APIs to ensure nodes can be upgraded with minimal disruption to running DC/OS services on a node. The upgrade procedure has also been simplified to improve user experience.
+
+For more information, see the [documentation](/1.9/installing/ent/upgrading/).
+
+# <a name="known-issues"></a>Known Issues and Limitations
+
+- DCOS_OSS-691 - DNS becomes unavailable during DC/OS version upgrades.
+- DCOS-14005 - Marathon-LB does not support pods.
+- DCOS-14021 - [Task logging to journald](/1.9/monitoring/logging/) disabled by default, so task logs will continue to be written to their sandboxes, and logrotated out. The `dcos task log` command will work as it did before.
+- DCOS-16737 - You cannot [generate and publish AWS Advanced Templates](/1.9/installing/ent/cloud/aws/advanced/#create-your-templates) to AWS GovCloud regions. When running the command `dcos_generate_config.ee.sh --aws-cloudformation` with GovCloud credentials you will see an error similar to this:
```

Comment:
Enterprise: add [enterprise type="inline" size="small" /]

Reply:
Added Enterprise tags.

Reply:
I'm not seeing the Enterprise tag on my build for this one 🤔